karma.conf.js

```diff
@@ -35,6 +35,12 @@ module.exports = function(config) {
             },
 
             {
+                pattern: 'test/lib/pptm.js',
```

Comment:
This is something else, the idea is to setup karma to serve this empty pptm.js file. So there's no 404 when the tests run, and the script is added to the DOM.